nci-crdc-demo.datacommons.io/manifest.json

```diff
@@ -14,15 +14,16 @@
     "aws-es-proxy": "abutaha/aws-es-proxy:0.8",
     "fence": "quay.io/cdis/fence:2.6.0",
     "indexd": "quay.io/cdis/indexd:1.1.0",
-    "peregrine": "quay.io/cdis/peregrine:1.0.5",
+    "peregrine": "quay.io/cdis/peregrine:1.0.9",
     "pidgin": "quay.io/cdis/pidgin:1.0.0",
     "revproxy": "quay.io/cdis/nginx:1.15.5-ctds",
-    "sheepdog": "quay.io/cdis/sheepdog:1.1.7",
-    "portal": "quay.io/cdis/data-portal:2.4.3",
+    "sheepdog": "quay.io/cdis/sheepdog:1.1.8",
+    "portal": "quay.io/cdis/data-portal:2.7.2",
```

Comment:
this portal version is a breaking change if the configuration hasn't been updated - will need to check this before merging

Reply:
@qingyashu mentioned since this portal is not gitops. We are good to go?